report_generator/outputs/full_report.md

```diff
@@ -0,0 +1,265 @@
+
+
+
+
+
+## reentrancy-eth
+
+* **Impact:** `High`
+* **Confidence:** `Medium`
+
+### In Function
+
+there should some sort of a code snapshot
+
+### Lines of Relevance
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+
+
+
+
+
+## divide-before-multiply
+
+* **Impact:** `Medium`
+* **Confidence:** `Medium`
+
+### In Function
+
+there should some sort of a code snapshot
+
+### Lines of Relevance
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+
+
+
+
+
+## naming-convention
+
+* **Impact:** `Informational`
+* **Confidence:** `High`
+
+### In Function
+
+there should some sort of a code snapshot
+
+### Lines of Relevance
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+
+
+
+
+
+## low-level-calls
+
+* **Impact:** `Informational`
+* **Confidence:** `High`
+
+### In Function
+
+there should some sort of a code snapshot
+
+### Lines of Relevance
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## MyThrill
+
+**Total number of issues:** 42069
+
+**Version of the tool:** 1.54.33
+
+
+
+
+
+
+
+
+
+## SMTChecker
+
+**Total number of issues:** 42069
```

Comment:
Hmm, SMTChecker and Mythrill find the exact same amount of issues and have the same version nr? 
Sounds like something went wrong here and we have a duplicate

Reply:
As noted in the other comment. This is indeed a POC.